src/main/java/lottogame/domain/Rank.java

```diff
@@ -0,0 +1,36 @@
+package lottogame.domain;
+
+import java.util.Arrays;
+
+public enum Rank {
+    FIRST(Rank.FIRST_PLACE, 2000000000),
```

Comment:
언더스코어(`_`)를 활용하여 단위가 큰 숫자들의 가독성을 높여볼 수 있을 것 같습니다.
```suggestion
    FIRST(Rank.FIRST_PLACE, 2_000_000_000),
```